libjolie/src/jolie/lang/nativeTypes/NativeType.java

```diff
@@ -28,45 +28,80 @@
  *
  * @author Fabrizio Montesi
  */
-public enum NativeType
-{
-	//UNDEFINED( "undefined" ),
-	STRING( "string" ),
-	INT( "int" ),
-	LONG( "long" ),
-	BOOL( "bool" ),
-	DOUBLE( "double" ),
-	VOID( "void" ),
-	RAW( "raw" ),
-	ANY( "any" );
-
-	private final static Map< String, NativeType > idMap = new HashMap<>();
-
-	static {
-		for( NativeType type : NativeType.values() ) {
-			idMap.put( type.id(), type );
-		}
+public class NativeType {
+
+	String refinement = "";
```

Comment:
I don't think that refinement should live here.

It should be a member of StringType, since for now we use it only there.